### ISSUE TITLE
Two LD1125H in one ESP32?

### ISSUE THREAD

**Issue by jcastro** [patrick3399/Hi-Link_mmWave_Radar_ESPHome#18]
Hey guys!

Have you tried to run 2 LD1125H sensors into one ESP32 module? Just curious to hear about your experience. Thanks!

**Comment by patrick3399** [comment]
You can test it yourself.
Suggestion use the hardware UART.

**Comment by jcastro** [comment]
For sure! Probably will test it today and report back

**Comment by jcastro** [comment]
IT WORKS! I will leave it a few days to see if it's stable and report back

**Comment by jcastro** [comment]
Yup! Really stable so far, but looks like putting 2 sensors side by side have some interferences

**Comment by patrick3399** [comment]
Nice. Because I only have one LD1125H, so hard to test duo radar at same ESP device.

**Comment by jcastro** [comment]
> 

Would love to donate some sensors if it’s helpful for you to test or add support (I heard there’s a new LD2450!)

**Comment by patrick3399** [comment]
> > 
> 
> Would love to donate some sensors if it’s helpful for you to test or add support (I heard there’s a new LD2450!)

Actually I already have LD2450 and LD2420 model, but there is no hurry to develope the code because we have few model to properly work.
